vancouver-tree-height-geo.Rmd

```diff
@@ -120,57 +115,64 @@ heatmap_data <- cont_table |>
     values_to = "Tree_Count"
   ) |> 
   mutate(
-    HEIGHT_RANGE_ID = factor(HEIGHT_RANGE_ID, levels = as.character(0:10))
+    HEIGHT_RANGE_ID = factor(HEIGHT_RANGE_ID,
+                             levels = as.character(0:10))
   )
 
-ggplot(heatmap_data, aes(x = HEIGHT_RANGE_ID, y = NEIGHBOURHOOD_NAME, fill = Tree_Count)) +
+ggplot(heatmap_data, aes(x = HEIGHT_RANGE_ID,
+                         y = NEIGHBOURHOOD_NAME,
+                         fill = Tree_Count)) +
   geom_tile() +
-  scale_fill_gradient(low = "white", high = "blue") +
+  scale_fill_gradient(low = "white",
+                      high = "blue") +
   labs(
-    title = "Tree Distribution by Neighbourhood and Height Range",
-    x = "Height Range",
+    title = "Figure 1: Tree Height Distribution by Neighbourhood",
+    x = "Height Category ID",
     y = "Neighbourhood",
     fill = "Tree Count"
   ) +
   theme_minimal() +
   theme(axis.text.x = element_text(angle = 45, hjust = 1))
 ```
 
-From the plot, it looks like there may be some significant differences in tree heights across neighbourhoods. In particular:
+From the plot, it looks like there may be some differences in tree heights across neighbourhoods. In particular:
 
--   sd
+-   RENFREW-COLLINGWOOD has, by a large margin, the most trees in the 10’ - 20’ height category (ID 1)
 
--   sd
+-   VICTORIA-FRASERVIEW, SUNSET, RENFREW-COLLINGWOOD, KENSINGTON-CEDAR COTTAGE, and HASTINGS-SUNRISE seem to have more trees in the lower height categories (1-4) than other neighbourhoods.
+
+-   SHAUGHNESSY and DOWNTOWN seem to have the tallest trees.
 
 ### Statistical Test
 
-is it statistically significant?
+Although there are some visually identifiable patterns, we can only determine whether there are statistically significant differences in tree heights per neighbourhood using an appropriate test.
 
-#### Choosing the Appropriate Test
+#### Choosing a Test and Significance Level
 
-Because we are dealing with two categorical variables each with multiple levels, a Chi-squared test of Independence/Homogeneity is appropriate
+Because we are dealing with two categorical variables, each with multiple levels, a Chi-squared test of Independence/Homogeneity is appropriate.
 
 The Chi-squared test makes the following assumptions:
 
 1.  The observations are independent.
-2.  The expected counts are sufficiently large (greater than 5 is the typical standard).
+2.  The expected counts are large enough (greater than 5 is typical).
+
+First, the height measurement of a particular tree does not depend on that of another, so we can assume independence. Second almost every intersection of the contingency table has counts much larger than 5, with many in the hundreds or thousands.
```

Comment:
You can add a comma behind the word "Second" to improve readability.

Reply:
Done